### ISSUE TITLE
CentOS 8 and Nginx version

### ISSUE THREAD

**Issue by virtualos** [engintron/engintron#1291]
Hello,

I'v installed CentOS 8 with new cPanel RC v94.0.1

Strange thing is Engintron always shows Nginx version as 1.14.1

I tried to update nginx mainline/stable versions but engintron shows nginx version number = 1.14.1

![Screenshot 2021-02-28 113921](https://user-images.githubusercontent.com/62283634/109411777-a50dfc80-79b9-11eb-8fe6-5a9b6fa7da3b.png)

**Comment by fevangelou** [comment]
Whats does `nginx -v` show in the terminal?

**Comment by virtualos** [comment]
> Whats does `nginx -v` show in the terminal?

nginx version: nginx/1.14.1

**Comment by fevangelou** [comment]
Then that's the installed version. Perhaps it's not updated by Nginx given the CentOS 8 will be dropped by Red Hat soon and CentOS Stream probably won't be an upgrade from CentOS 8...

**Comment by virtualos** [comment]
> Then that's the installed version. Perhaps it's not updated by Nginx given the CentOS 8 will be dropped by Red Hat soon and CentOS Stream probably won't be an upgrade from CentOS 8...

So it means I should better stick to centos7/cloudlinux7

**Comment by virtualos** [comment]
> Then that's the installed version. Perhaps it's not updated by Nginx given the CentOS 8 will be dropped by Red Hat soon and CentOS Stream probably won't be an upgrade from CentOS 8...

Hi again,

Just for your info:

I installed CentOS 8 (I did "yum update") then cPanel RC v94.0.1 after that converted to Cloudlinux 8(They claim that end of life is May 31, 2029) and finally Engintron, Hope to see nginx update but still no luck, version in stable/mainline **1.14.1**

I can share with you this vps if you want to see...

**Comment by fevangelou** [comment]
Switch to mainline. If that does not bump you to v1.19.x for Nginx, then manually install the latest stable version using the relevant rpm file: `yum install http://nginx.org/packages/centos/8/x86_64/RPMS/nginx-1.18.0-2.el8.ngx.x86_64.rpm` (or just `rpm -ivh http://nginx.org/packages/centos/8/x86_64/RPMS/nginx-1.18.0-2.el8.ngx.x86_64.rpm` if yum fails for some reason).

**Comment by virtualos** [comment]
Hello,

I should reopen this issue. I upgraded my vps from cloudlinux 7 to cloudlinux 8 and I have no luck to update nginx from 1.14 to more recent versions as you offer, I have to say I tried some ways to manually install nginx but I encounter with Welcome to nginx page for all my customers account.

I believe it's time for Engintron to officially support Centos 8 and Cloudlinux 8 and test against all bugs and limitations as we put centos6/7 cloudlinux 6/7 behind.

Right now I'm using cpanel 96 nginx core with cloudlinux 8 but I prefer Engintron.
